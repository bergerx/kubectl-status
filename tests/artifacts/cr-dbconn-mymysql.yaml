apiVersion: example.com/v1alpha1
kind: DatabaseConnection
metadata:
  creationTimestamp: "2024-07-25T19:58:20Z"
  name: mymysql
  namespace: default
  resourceVersion: "1330976"
  uid: be814fad-43ba-4d86-bc5a-f4fb471af6ae
spec:
  defaultSchema: myapp
  engine: mysql
  rootPassword: password123
  rootUser: admin
status:
  observedGeneration: 1
  conditions:
  - lastTransitionTime: "2024-07-25T15:27:00Z"
    message: Database connection successfully established.
    reason: ConnectionEstablished
    status: "True"
    type: Ready
