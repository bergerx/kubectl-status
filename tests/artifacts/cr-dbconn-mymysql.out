
DatabaseConnection/mymysql -n default, created 1m ago
  Current: Resource is Ready
  Ready ConnectionEstablished, Database connection successfully established. for 1m
